test-utils/src/lib.rs

```diff
@@ -293,3 +297,34 @@ pub fn build_expected_hash(values: &[u64]) -> [Felt; 4] {
 
     expected
 }
+
+/// Mutates a seed and generates a word deterministically
+#[cfg(not(target_family = "wasm"))]
+pub fn seeded_word(seed: &mut [u8; 32]) -> Word {
+    [
+        seeded_element(seed),
+        seeded_element(seed),
+        seeded_element(seed),
+        seeded_element(seed),
+    ]
+}
```

Comment:
I'm not sure we need this function. We can probably just do:
```Rust
let x: Word = rand::prng_array(seed);
```

Reply:
We need to at least mutate the seed - otherwise, we will generate the same word

```rust
let seed = [0xff; 32];
let a: Word = test_utils::rand::prng_array(seed);
let b: Word = test_utils::rand::prng_array(seed);
assert_eq!(a, b);
```

`seeded_word` is a convenience/helper function to mutate the seed and generate the word output. However, it makes more sense to use the `rand::prng_array` for fetch the word, after we mutated the seed. Fixed!

Reply:
Ah - got it. A couple of comments on this:

1. I think these functions should go into `rand` module. For this, we might have to crate a wrapper module and then re-export other functions from `rand-utils` from there.
2. Should we simplify these to take seed as a `u64` value? This way, we can just increment the seed to update it and then use the 8 bytes to initialize the 32-byte array. This would be simpler/more efficient, and for test purposes should provide enough entropy.

Reply:
That makes sense! The `u64` will be as good as any sequence of bytes as the cryptographic RNG will guarantee us uniform bits